src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/SscsBulkScanFunctionalTest.java

```diff
@@ -114,7 +114,8 @@ private Response simulateCcdCallback(String json, String urlPath) {
 
     private void createCase() {
         SscsCaseData caseData = CaseDataUtils.buildMinimalCaseData();
-        SscsCaseDetails caseDetails = ccdService.createCase(caseData, idamTokens);
+        SscsCaseDetails caseDetails = ccdService.createCase(caseData, "appealCreated",
+            "Bulk Scan appeal created", "Bulk Scan appeal appeal created in test", idamTokens);
```

Comment:
minor point but the word appeal appears twice

Reply:
corrected. thanks.